_scripts/deploy.sh

```diff
@@ -6,7 +6,7 @@
 cd "$(dirname "$0")" || exit 1
 
 export IMAGE_PREFIX=deisci BUILD_TAG=v2-beta
-docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
-DEIS_REGISTRY='' make -C .. build push
+#docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
```

Comment:
The general strategy for other components is to push Docker images to both Docker Hub and quay.io, so we shouldn't comment this out unless there's a good reason.